service/src/intTest/resources/application.yml

```diff
@@ -40,13 +40,11 @@ gp2gp:
     ttl: ${GP2GP_MONGO_TTL:P7D}
     cosmosDbEnabled: ${GP2GP_COSMOS_DB_ENABLED:false}
   gpc:
-    url: ${GP2GP_GPC_GET_URL:https://orange.testlab.nhs.uk/B82617/STU3/1/gpconnect}
+    url: https://orange.testlab.nhs.uk/B82617/STU3/1/gpconnect
```

Comment:
Does removing the environment variable mean our Jenkins builds now use the public demonstrator instead of wiremock?

Reply:
Yes, unfortunately if we set GP2GP_GPC_GET_URL to gpcc url the tests freeze for some reason.

Reply:
Yes

Reply:
We must ensure the pipeline continues to use wiremock. The default for local development should be orange testlab.